src/Window.cpp

```diff
@@ -293,6 +293,10 @@ Window::Window(QWidget *parent)
   refreshMazeFileComboBox(path);
   updateMazeAndPath(maze, path);
 
+  QStringList SplitStr = path.split("/");
+  QString TitleStr = SplitStr.takeLast();
+  this->setWindowTitle("mms : "+TitleStr);
+
```

Comment:
This looks like an unrelated change. Can you split it out into its own pull request? Same goes for similar changes below.